cli/command_repository_upgrade.go

```diff
@@ -0,0 +1,240 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/kopia/kopia/internal/epoch"
+	"github.com/kopia/kopia/repo"
+	"github.com/kopia/kopia/repo/content"
+)
+
+type commandRepositoryUpgrade struct {
+	forceRollback   bool
+	skip            bool
+	blockUntilDrain bool
+
+	// lock settings
+	advanceNoticeInterval  time.Duration
```

Comment:
I see we assign`UpgradeLock{AdvanceNoticeDuration: ...}` to this field's value. Should we name this field accordingly? (more changes needed for suggestion).
```suggestion
	advanceNoticeDuration  time.Duration
```